types/whatwg-url/lib/URL-impl.d.ts

```diff
@@ -0,0 +1,19 @@
+declare class URLImpl {
```

Comment:
I'm not sure we want typings for this or any of the other lib/impl files. They seem like implementation details, don't they? E.g. I would think the URL constructor would always be imported directly from `index.js`, and so including typings for the implementation would only help people shoot themselves in the foot since the implementation may change.

Reply:
Well, I&nbsp;was&nbsp;allowed to&nbsp;include the&nbsp;`DOMExceptionImpl`&nbsp;class in&nbsp;<https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41390>.

Reply:
Have you actually found a need for `DOMExceptionImpl` or other internal libs/classes which is driving adding the types? I'm not _positive_ we shouldn't have them here if there is some use I'm missing, but I suspect we don't need them.

Reply:
The reason is future interop plans in <https://github.com/jsdom/webidl2js/issues/69>.